src/components/Karma/CircleGradient.tsx

```diff
@@ -47,12 +48,16 @@ export const CircleGradient = (props: {
       </LinearGradient>
       {props.fill !== 100 && (
         <Svg
-          height={props.size}
-          width={props.size}
-          style={{position: 'absolute', transform: [{rotateZ: '-90deg'}]}}>
+          height={props.size + Math.ceil(props.width)}
+          width={props.size + Math.ceil(props.width)}
+          viewBox={`0 0 ${props.size} ${props.size}`}
+          style={{
+            position: 'absolute',
+            transform: [{rotateZ: '-90deg'}]
+          }}>
           <Circle
-            cx={props.size / 2}
-            cy={props.size / 2}
+            cx={props.size / 2 + 0.2}
+            cy={props.size / 2 - 1.7}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The viewBox attribute has been set to `0 0 ${props.size} ${props.size}` and the center coordinates of the circle have been slightly modified. These changes will affect the viewable area and the position of the circle within the SVG. Ensure these changes align with your design expectations.
```diff
- viewBox={`0 0 ${props.size} ${props.size}`}
- cx={props.size / 2}
- cy={props.size / 2}
+ viewBox={`0 0 ${props.size + Math.ceil(props.width)} ${props.size + Math.ceil(props.width)}`}
+ cx={props.size / 2 + 0.2}
+ cy={props.size / 2 - 1.7}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->